app/views/includes/scripts.html

```diff
@@ -10,4 +10,8 @@
 
 {% if enableTracking === 'true' %}
 <script src="/public/javascripts/gatracking.js"></script>
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+<!--[if IE 8]>
+<script src="/public/javascripts/ie8polyfill.js"></script>
```

Comment:
Perhaps a better name for this file would be something like `form-polyfill.js` as it's then linked to the functionality and not then tied to a specific browser?